settings.php

```diff
@@ -1073,13 +1073,13 @@ function convertseconds($argument)
                             <div class="col-lg-6 col-md-12">
                                 <div class="box box-warning">
                                     <div class="box-header with-border">
-                                        <h3 class="box-title">Teleporter Export</h3>
+                                        <h3 class="box-title">Teleporter Backup</h3>
```

Comment:
Feel free to remove the word "Teleporter" completely.

Reply:
Agreed.